spec/factories.rb

```diff
@@ -1,21 +1,15 @@
 FactoryGirl.define do
-  factory :sith_user, class: User do
+  factory :user, class: User do
     sequence :email do |n|
       "kylolovesgrandpa#{n}@example.com"
     end
     password "finishwhatyoustarted"
     password_confirmation "finishwhatyoustarted"
   end
-
-  factory :jedi_user, class: User do
-    sequence :email do |n|
-      "rey@example.com"
-    end
-    password "wherearemyparents"
-    password_confirmation "wherearemyparents"
-  end
   
   factory :game do
+    sith_user_id 1
+    jedi_user_id 2
```

Comment:
These lines above I don't think are doing what you expect.  I think they should be removed.